### PR TITLE
Add /usr/local/lib{64} to /etc/ld.so.conf by default

### LINKED ISSUE #26: Add standard build directories to LD_LIBRARY_PATH?
Should we add `/usr/local/lib:/usr/local/lib64`to `LD_LIBRARY_PATH` by default?

I notice openblas, for example, builds to `/usr/loca/lib`.

### PR DESCRIPTION
**Pull request by ogrisel**
Fix #26.

### CODE REVIEW

**Comment by ogrisel** [comment]
You will probably need to run `ldconfig` in your scripts after you install libraries to `/usr/local/lib`.

File: docker/build_scripts/build.sh [hunk 1]
```diff
@@ -23,6 +23,10 @@ MANYLINUX1_DEPS="glibc-devel libstdc++-devel glib2-devel libX11-devel libXext-de
 MY_DIR=$(dirname "$BASH_SOURCE[0]}")
```
Comment (piotr-dobrogost) [inline docker/build_scripts/build.sh]: Is left curly bracket missing here?

Comment (ogrisel) [inline docker/build_scripts/build.sh]: Indeed. It's weird that bash does not complain in this case. Will push a fix.

**Comment by njsmith** [comment]
Would setting LD_LIBRARY_PATHS in the Dockerfile instead avoid the need for rerunning ldconfig? That seems like the kind of tripwire that will catch a lot of people...

**Comment by njsmith** [comment]
We also might want to canonize Matthew's PKGCONFIG_PATH change: https://github.com/matthew-brett/manylinux-builds/blob/master/common_vars.sh#L16

**Comment by ogrisel** [comment]
@njsmith alright I did the change.

**Comment by njsmith** [comment]
@Matthew-Brett: this should let you clean up those scripts a bit, let us know if it doesn't work for some reason :-)

**Comment by matthew-brett** [comment]
It works, thanks - see : https://github.com/matthew-brett/manylinux-builds/commit/d439645180f5beff446972e22b672b6e6c4a3b65

**Pull request merged**